src/js/profile/mobile/widget/Calendar.js

```diff
@@ -0,0 +1,835 @@
+
+										if (changeYear1Week < weekCount && changeYear1Week > 1) {
+											weekCount = 1;
+										}
+									} else {
+										if (weekCount > 52 && this._activeMonth.getMonth() == 11) {
+											var lastWeekCount = this.getWeekNumber(new Date(activeDate.getFullYear() + 1, 0, 1));
+
+											if (lastWeekCount == 1) {
+												weekCount = 1;
+											}
+										}
+									}
+									this.contDates.childNodes[oneWeek].childNodes[dayCount].innerHTML = "<div class='calendar_label'>" + weekCount + "</div>";
+								} else {
+									var activeMonthAllDays = new Date(activeDate.getFullYear(), activeDate.getMonth(), activeDate.getDate() + addMonthCount, 0, 0, 0, 0);
+
+									if (activeMonthAllDays.getHours() != 0) {
+										var meridiem = (activeMonthAllDays.getHours() > 12 ? 24 - activeMonthAllDays.getHours() : activeMonthAllDays.getHours());
+
+										activeMonthAllDays.setTime(activeMonthAllDays.getTime() + 60 * 60 * 1000 * meridiem);
+									}
+									var allDaysToTime = activeMonthAllDays.getTime();
+
+									this.contDates.childNodes[oneWeek].childNodes[dayCount].innerHTML = "<div class='calendar_label'>" + activeMonthAllDays.getDate() + "</div>";
+									this.contDates.childNodes[oneWeek].childNodes[dayCount]._date = new Date(allDaysToTime);
+									this.contDates.childNodes[oneWeek].childNodes[dayCount]._hLocation = oneWeek;
+									this.contDates.childNodes[oneWeek].childNodes[dayCount]._vLocation = dayCount;
+									this.contDates.childNodes[oneWeek].childNodes[dayCount].cssMonth = (activeMonthAllDays.getMonth() == activeMonth);
+									this.contDates.childNodes[oneWeek].childNodes[dayCount].cssDate = (!this._nullDate && allDaysToTime == timeToNumber);
+									this.contDates.childNodes[oneWeek].childNodes[dayCount].cssWeekend = (weekStart >= 7);
+									this._updateCellStyle(oneWeek, dayCount, false);
+									if (allDaysToTime == timeToNumber) {
+										this._activeDateCell = this.contDates.childNodes[oneWeek].childNodes[dayCount];
+										this._updateCellStyle(oneWeek, dayCount, true);
+									}
+									if (++weekStart > 7) {
+										weekStart = 1;
+									}
+									addMonthCount++;
+								}
+							}
+						}
+						this.contMonth.firstChild.firstChild.childNodes[1].innerHTML = this._buildMonthHdr(dateFullFormat);
+					};
+					this._updateCellStyle = function (col, row, flag) {
+						var selectDate = this.contDates.childNodes[col].childNodes[row];
+						var cellStr = "calendar_cell calendar_cell";
+
+						cellStr += (selectDate.cssMonth ? "_month" : "");
+						cellStr += (selectDate.cssDate ? "_date" : "");
+						cellStr += (selectDate.cssWeekend ? "_weekend" : "");
+						selectDate.className = cellStr;
+						var child = selectDate.getElementsByTagName("div")[0];
+
+						if (flag === false) {
```

Comment:
How about the code for style to move in .less file?

Reply:
Styles that change on condition cannot be applied to less.